_posts/2021-09-07-emergent-practices.md

```diff
@@ -0,0 +1,96 @@
+
+Following the individual presentations of these projects, we opened up the discussion to dig deeper into these topics. Here were some of the key highlights:
+
+## Money as a means, not an end
+Alanna observed how many people have a complicated relationship with money, and that talking about it can raise some big feelings. Society, she said, has ceded a lot of power around money to the extractive mainstream economy. Because of this, many community projects cringe and move away from using money as effectively as they could. She thinks that those in the alternative solidarity space should instead think of money as a way to make things happen. Dawn commented that money can expand community capacity, and be a tool for creating accountability for tasks and projects to get done. Money can also shed light on the real sustainability of a project. Without enough of it to compensate people, projects end up having to rely on free labor — a privilege that many people cannot afford.
+
+  > “We need to reclaim the power of money, use it differently, and direct it towards pro-social values.” —Alanna Irving
+
+## Make invisible labor visible, and valued
+We discussed how enabling people to secure a livelihood with their project does not just create more resiliency, it can be a way to make visible the types of the undervalued labor that too often goes undercompensated. Especially with tech-focused projects, all the work that goes into ensuring the tool or platform is successful is usually not as rewarded as the work of building the tool itself. That includes writing documentation, onboarding new contributors and end-users, and publicizing the product. A radical approach to money can get us to the root of these dynamics, and brings these all-to-prevalent labor inequities to the fore.
+
+## Addressing the accessibility of tech tools
+Across the board, there was an agreement that many of the tools we build and use remain inaccessible to many. Scott acknowledged how there’s still a lot of work to be done in the cryptocurrency ecosystem, as well as in open source software where major technical contributors are prioritized over individual members of the community. We then discussed how with greater cooperation, tooling, and patterns of working together, that the tech side of projects could just be another specialization, as a role that could be designated to someone. This was part of a greater recognition that we need more diversity within projects to specialize in different aspects of an organization. Tools like Open Collective and organizations like Hypha Co-operative help remove the financial, legal, and administrative barriers for those trying to build their project or address frontline needs, by enabling them to outsource this work.
+
+~
```

Comment:
This as a text mark doesn't really flow with text
```suggestion

```

Reply:
see latest